pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -748,7 +748,7 @@ func (c *Cluster) getPVCHostName(pvcName string) (string, error) {
 		if name == "" {
 			return "", errors.Errorf("node name not found on the osd pod %q", pod.Name)
 		}
-		return name, nil
+		return name, nil //nolint
```

Comment:
An explanation of nolint would be helpful
```suggestion
		return name, nil //nolint: no need for else statement
```

Reply:
changing to `//nolint, no need for else statement`, using `:` will give an error because it is used to specify specific linter.